tests/layer_tests/pytorch_tests/test_floor_divide.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import numpy as np
+from pytorch_layer_test_class import PytorchLayerTest
+
+
+class Testfloor_divide(PytorchLayerTest):
+    def _prepare_input(self):
+        return (self.input_tensor, self.other_tensor)
+
+    def create_model(self):
+
+        import torch
+
+        class aten_floor_divide(torch.nn.Module):
+            def __init__(self):
+                super(aten_floor_divide, self).__init__()
```

Comment:
Why don't you simply use `super().__init__()` ? Probably it is not a problem, but I am curious ;)

Reply:
Just a convention that I am used to :)